hack/verify-md.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+readonly CURRENT_DIR="$( cd "$( dirname "$0" )" && pwd )"
+readonly ROOT_PATH="$( cd "${CURRENT_DIR}/../" && pwd )"
+
+TMP_DIR=$(mktemp -d)
+
+cleanup() {
+    rm -rf "${TMP_DIR}" || true
+}
+
+trap cleanup EXIT SIGINT
+
+cp -a ${ROOT_PATH}/docs/05-technical-reference/00-custom-resources/. $TMP_DIR/
+
+cd $CURRENT_DIR/table-gen || exit
+
+go run $CURRENT_DIR/table-gen/table-gen.go --crd-filename ../../installation/resources/crds/telemetry/tracepipelines.crd.yaml --md-filename $TMP_DIR/telemetry-03-tracepipeline.md --crd-title TracePipeline
```

Comment:
Why the mapping of the files gets repeated here? Shouldn't we just call the Makefile? We should have only one place where we map a CRD to a MD file, otherwise we will forget to update both always